data/README.md

```diff
@@ -0,0 +1 @@
+Download the [data](https://www.kaggle.com/datasets/snap/amazon-fine-food-reviews) from kaggle. It's too big (~256 MB) to upload to github.
```

Comment:
Since we are not using the whole dataset you should be able to upload the whole file.

Reply:
At this point not sure, which reviews would be used especially in the case of testing. In some cases, I am picking any random data out of the samples (20/40) fed.
So, it would be better to download manually from the given link.

Reply:
I think you can still add the file as it is used now and note that it comes from a bigger dataset with the link. This will make setup much better.

Reply:
Ok I will trim it & upload then.

Reply:
Added in this commit: https://github.com/abhi3700/semantic-hashing-demo/commit/b2245654fa144366eb2e2ef9e6232497a864759a